src/interpolation/compute/formatters.ts

```diff
@@ -0,0 +1,14 @@
+import { Formatters } from '../../types'
+
+const defaultFormatters: Formatters = {
+  lowercase: (s: string) => s.toLowerCase(),
+  uppercase: (s: string) => s.toUpperCase(),
+  capitalize: (s: string) =>
+    s
+      .split(' ')
+      .map(([head, ...tail]) => head.toUpperCase() + tail.join('').toLowerCase())
+      .join(' '),
+  money: (s: string, round: number, currency: string) => parseInt(s).toFixed(round) + currency
```

Comment:
Good args, but money formatting is pretty complex, I would suggest to use [toLocaleString()](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Number/toLocaleString) instead to follow standards